src/openzeppelin/account/library.cairo

```diff
@@ -225,21 +223,55 @@ namespace Account:
         ) -> (response_len: felt, response: felt*):
         alloc_locals
 
-        let (caller) = get_caller_address()
-        with_attr error_message("Account: no reentrant call"):
-            assert caller = 0
-        end
-
         let (__fp__, _) = get_fp_and_pc()
         let (tx_info) = get_tx_info()
 
         # validate transaction        
-        let (is_valid) = is_valid_eth_signature(tx_info.signature_len, tx_info.signature)
-        with_attr error_message("Account: invalid signature"):
+        let (is_valid) = is_valid_eth_signature(tx_info.transaction_hash,tx_info.signature_len, tx_info.signature)
```

Comment:
```suggestion
        let (is_valid) = is_valid_eth_signature(tx_info.transaction_hash, tx_info.signature_len, tx_info.signature)
```

Reply:
same as with the vanilla implementation, this line should be within the `with_attr` block so it's catched as per [@juniset's comment](https://github.com/OpenZeppelin/cairo-contracts/pull/361#discussion_r899845945)